toqito/matrices/hadamard.py

```diff
@@ -20,28 +20,25 @@ def hadamard(n_param: int = 1) -> np.ndarray:
 
     .. math::
         \left( H_n \right)_{i, j} = \frac{1}{2^{\frac{n}{2}}}
-        \left(-1\right)^{i \dot j}
+        \left(-1\right)^{i \cdot j}
 
     Examples
     ==========
 
-    The standard 2-qubit Hadamard matrix can be generated in :code:`toqito` as
+    The standard 1-qubit Hadamard matrix can be generated in :code:`|toqito⟩` as
```

Comment:
I am fine with the 2 -> 1 qubit change, the other change is already in the main branch...